config/environments/development.rb

```diff
@@ -59,5 +59,13 @@ def filter(msg)
   config.assets.debug = true
   config.react.variant = :development
   config.active_record.logger = nil
+  #Line 63-69 are for 'bullet' gem initialization.
+  config.after_initialize do
+    Bullet.enable = true
+    Bullet.alert = true
+    Bullet.bullet_logger = true
+    Bullet.console = true
+    Bullet.rails_logger = true
```

Comment:
do we need to be logging these when we merge this project in?

Reply:
We won't need logging when we merge. I will turn all these to 'false'. This configuration is for 'bullet gem', which helps in detecting N+1 problems in the application using logging. This can be later turned to 'true' if someone wishes to find and resolve N+1 issues in future codebase.